### PR TITLE
Created digital_ocean_block_storage module

### PR DESCRIPTION
**Pull request by harneksidhu**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

digital_ocean_block_storage
##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file =
  configured module search path = Default w/o overrides
```
##### SUMMARY

This PR is submitted because of the #4200 feature request and the DigitalOcean rework #4219.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @harneksidhu for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by alukovenko** [comment]
shipit

**Comment by harneksidhu** [comment]
@kontrafiktion shipit?

**Comment by harneksidhu** [comment]
@gregdek  can this be merged?

File: cloud/digital_ocean/digital_ocean_block_storage.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    description:
+    - The timeout in seconds used for polling DigitalOcean's API.
+    default: 10
+
+notes:
+  - Two environment variables can be used, DO_API_KEY and DO_API_TOKEN.
+    They both refer to the v2 token.
+
+author:
+    - "Harnek Sidhu (github: harneksidhu)"
```
Comment (gundalow) [inline cloud/digital_ocean/digital_ocean_block_storage.py]: Minor comment, but can you please change this to:
"Harnek Sidhu (@harneksidhu)"

In the future we may be parsing these lines

**Review by gundalow — COMMENTED** [review]
One minor comment, rest looks good.

File: cloud/digital_ocean/digital_ocean_block_storage.py (hunk 1 above)
Comment (harneksidhu) [reply cloud/digital_ocean/digital_ocean_block_storage.py]: @gundalow Fixed

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**